docs/Documentation.md

```diff
@@ -1,29 +1,39 @@
----
-layout: page
-title: Documentation guide
----
+<frontmatter>
+  layout: default.md
+  title: "Documentation guide"
+  pageNav: 3
+  pageNavTitle: "Table Of Content"
+</frontmatter>
+
+# Documentation guide
 
 **Setting up and maintaining the project website:**
 
-* We use [**Jekyll**](https://jekyllrb.com/) to manage documentation.
+* We use [**MarkBind**](https://markbind.org/) to manage documentation.
 * The `docs/` folder is used for documentation.
-* To learn how set it up and maintain the project website, follow the guide [_[se-edu/guides] **Using Jekyll for project documentation**_](https://se-education.org/guides/tutorials/jekyll.html).
+* To learn how set it up and maintain the project website, follow the [_**MarkBind User Guide**_](https://markbind.org/userGuide/gettingStarted.html).
+  * Quick start:
```

Comment:
As this is reader's first encounter with MarkBind authoring, I prefer to be slightly more detailed in the Quick Start (so that she can get the site up and running locally in a few minutes), and point to the main MarkBind UG only at the end of it (to find more details). Note that some may not even have node installed.

Reply:
Just realized we also have this: https://se-education.org/guides/tutorials/markbind.html
which is not very up-to-date and applicable to the workflow [here](https://markbind.org/userGuide/gettingStarted.html#:~:text=Alternative%20installation%3A%20as%20a%20local%20dev%2Ddependency%20with%20package.json)

Do you think it will be good to
- update the se-edu guide and link it here, or
- add a specific page in markbind docs about project setup and link it here, or
- expand the details here

?

Reply:
@tlylt Here, we can give an AB3-specific trimmed-down guide so that the reader can get started on basic editing of AB3 docs quickly, without being distracted by extraneous details (i.e., there is no need to know about the `init` command). But we can point to MarkBind docs for additional details that the reader might not need right away, as we don't want too much duplication here either.  
I don't think we need to involve the se-edu/guide here, as it is not specific enough for AB3.
But if you spot any inaccuracies in the se-edu/guides version, pls let me know (or submit a PR).